Objects/Converters/ConverterCSI/ConverterCSIShared/ConverterCSIUtils.cs

```diff
@@ -27,18 +27,45 @@ public string ModelUnits()
       return _modelUnits;
     }
 
+#if ETABS || SAFE
+
+    _modelUnits = GetModelUnitsFromETABS();
+
+#else
```

Comment:
What about CSI Bridge?

Reply:
It's my understanding that SAP 2000 and CSiBridge are the same.:

<img width="283" alt="image" src="https://github.com/user-attachments/assets/19bdf332-54d9-4e1d-964a-17c5f81dba2f">

But I have requested a trial for CSiBrdige to get my hands on the documentation to confirm. Will keep you in the loop

Reply:
Ok that sounds good. I'm happy to merge without testing, but if you could check with @connorivy how we can test CSiBridge.
I'd like to do a quick test before we make the proper stable 2.20 release. But I'm not sure what licencing we have etc..